kotsadm/pkg/store/s3pg/reporting_store.go

```diff
@@ -126,3 +150,70 @@ func (s S3PGStore) getK8sVersion() (string, error) {
 	}
 	return k8sVersion.GitVersion, nil
 }
+
+func getKotsadmIDConfigMap() (*corev1.ConfigMap, error) {
```

Comment:
i don't think this function should exist in the store (where we retrieve data from postgres db). i would suggest moving this to https://github.com/replicatedhq/kots/blob/master/pkg/k8s/kotsadm.go